#! /bin/sh /usr/share/dpatch/dpatch-run
## h323_fix_1217.dpatch by Tzafrir Cohen <tzafrir.cohen@xorcom.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Upstream changeset 58008 broke h323 building. Revert it.
## DP: (Should probably be fixed by 1.2.18) 
## DP: Upstream issue: http://bugs.digium.com/view.php?id=9501

@DPATCH@
diff -urNad asterisk-1.2.17~dfsg~/channels/h323/Makefile asterisk-1.2.17~dfsg/channels/h323/Makefile
--- asterisk-1.2.17~dfsg~/channels/h323/Makefile	2007-03-06 17:17:16.000000000 +0200
+++ asterisk-1.2.17~dfsg/channels/h323/Makefile	2007-04-08 17:46:34.000000000 +0300
@@ -21,8 +21,7 @@
 OPENH323DIR=$(HOME)/openh323
 endif
 
-clean:
-	rm -f *.so *.o .depend
+-include $(OPENH323DIR)/openh323u.mak
 
 $(SOURCES)::	$(SOURCES:.cxx=.cpp)
 	ln -f $< $@
@@ -31,7 +30,6 @@
 	touch $(SOURCES)
 
 libchanh323.a:	$(OBJS)
-	include $(OPENH323DIR)/openh323u.mak
 	ar crv $@ $(OBJS)
 
 Makefile.ast:	FORCE
